torchax/test_dist/test_mesh_util.py

```diff
@@ -0,0 +1,31 @@
+import unittest
```

Comment:
Need to add this file into the unit test shell script and activate the CPU SPMD env var, I think

Reply:
it already taken care of here: https://github.com/pytorch/xla/blob/master/.github/workflows/torchax.yml#L59